test/main.test.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, expect, it, useCommand } from "./suite.ts";
+import { run, sleep, expect as $ } from "../mod.ts";
```

Comment:
nit: `$` is weird and confusing to read, especially with my past trauma from jquery; maybe this is the exposure therapy I need?

Reply:
Yeah, I'm not a super fan of it, and it was just there to get rid of the conflict for `expect()`

What do you think of this 😈 

https://github.com/thefrontside/effection/pull/717/files#diff-2608d3bd5af09d726fe4183133e684f22d2a52739a65f893e54a170bcabfa578R7-R16

Only appropriate for an end-user application like a test-suite (not a library) but it's nice in that it let's us yield* to promises directly.